### PR TITLE
Fix scrollTo deprecation warnings

### LINKED ISSUE #118: warning:  Use `scrollTo({x: 5, y: 5, animated: true})` instead.
It keeps showing 'warning: JavaScript] `scrollTo(y, x, animated)` is deprecated. Use `scrollTo({x: 5, y: 5, animated: true})` instead.' after I upgrade to rn 0.22.0. I hope you can fix this problem.

### PR DESCRIPTION
**Pull request by seb0zz**
This fixes #118, #106 and #95 by also considering the usage of `ViewPager` on Android.

### CODE REVIEW

**Comment by shaneosullivan** [comment]
It would also be good to make the 'animated' property available, so it can be optional.  There are cases where you'd just like to jump immediately to an index

**Comment by seb0zz** [comment]
@shaneosullivan you are right, just updated the PR

File: dist/index.js [hunk 1]
```diff
@@ -311,14 +311,22 @@ module.exports = _reactNative2.default.createClass({
    * @param  {number} index offset index
    */
   scrollTo: function scrollTo(index) {
```
Comment (shaneosullivan) [inline dist/index.js]: Can you also add the 'animated' parameter to the function signature, and document it above the function?

File: src/index.js [hunk 2]
```diff
@@ -310,15 +310,23 @@ module.exports = React.createClass({
...
     let diff = (this.props.loop ? 1 : 0) + index + this.state.index
     let x = 0
     let y = 0
     if(state.dir == 'x') x = diff * state.width
     if(state.dir == 'y') y = diff * state.height
-    this.refs.scrollView && this.refs.scrollView.scrollTo(y, x)
+    if (this.refs.scrollView) {
+      if (Platform.OS === 'ios') {
+        this.refs.scrollView.scrollTo({y, x, animated})
+      } else {
```
Comment (shaneosullivan) [inline src/index.js]: Perhaps use an explicit Android check here, given that Tizen and Windows support is coming?

**Comment by seb0zz** [comment]
@shaneosullivan made the android check explicit

**Comment by gwhite-dayspring** [comment]
@shaneosullivan any reason not to merge this?

**Comment by rajkissu** [comment]
Hey guys, this has been fixed in #112 - closing this pull request.

**Pull request closed without merging**